lib/network/modules/components/Node.js

```diff
@@ -168,6 +154,46 @@ class Node {
 
 
   /**
+   * Load the images from the options, for the nodes that need them.
+   *
+   * TODO: The imageObj members should be moved to CircularImageBase.
+   *       It's the only place where they are required.
+   *
+   * @private
+   */
+  _load_images() {
+    // Don't bother loading for nodes without images
+    if (this.options.shape !== 'circularImage' && this.options.shape !== 'image') {
+      return;
+    }
+
+    if (this.options.image === undefined) {
+      throw "Option image must be defined for node type '" + this.options.shape + "'";
+    }
+
+    if (this.imagelist === undefined) {
+      throw "No imagelist provided";
```

Comment:
This is a fair point. `this.images` comes from `Network.images`, which is an internal structure.

All the user needs to know at this point is that no images at all were defined. So, this should be sufficient:

```
  throw "No images provided";
```

I think this covers it; I'm open to other suggestions.